e2etest/test_manager.go

```diff
@@ -217,7 +217,7 @@ func tempDir(t *testing.T) (string, error) {
 		return "", err
 	}
 
-	if err = os.Chmod(tempPath, 0777); err != nil {
+	if err = os.Chmod(tempPath, 0o777); err != nil {
```

Comment:
it's the same - octal number. Some people say it's easier to read, but I don't have much opinion on that (it was fixed automatically by `make format`